workflow-examples/src/main/java/com/redhat/parodos/examples/move2kube/move2kubeWorkFlowConfiguration.java

```diff
@@ -0,0 +1,123 @@
+package com.redhat.parodos.examples.move2kube;
+
+import java.util.Arrays;
+import java.util.concurrent.Executors;
+
+import com.redhat.parodos.examples.move2kube.checker.TransformChecker;
+import com.redhat.parodos.examples.move2kube.task.GitArchiveTask;
+import com.redhat.parodos.examples.move2kube.task.GitBranchTask;
+import com.redhat.parodos.examples.move2kube.task.GitCommitTask;
+import com.redhat.parodos.examples.move2kube.task.Move2KubePlan;
+import com.redhat.parodos.examples.move2kube.task.Move2KubeRetrieve;
+import com.redhat.parodos.examples.move2kube.task.Move2KubeTask;
+import com.redhat.parodos.examples.move2kube.task.Move2KubeTransform;
+import com.redhat.parodos.tasks.git.GitCloneTask;
+import com.redhat.parodos.workflow.annotation.Checker;
+import com.redhat.parodos.workflow.annotation.Infrastructure;
+import com.redhat.parodos.workflow.consts.WorkFlowConstants;
+import com.redhat.parodos.workflows.workflow.ParallelFlow;
+import com.redhat.parodos.workflows.workflow.SequentialFlow;
+import com.redhat.parodos.workflows.workflow.WorkFlow;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class move2kubeWorkFlowConfiguration {
+
+	@Bean
+	GitCloneTask gitCloneTask() {
+		return new GitCloneTask();
+	}
+
+	@Bean
+	GitArchiveTask gitArchiveTask() {
+		return new GitArchiveTask();
+	}
+
+	@Bean
+	GitCommitTask gitCommitTask() {
+		return new GitCommitTask();
+	}
+
+	@Bean
+	GitBranchTask gitBranchTask() {
+		return new GitBranchTask();
+	}
+
+	private String getMove2KubeAPIEndpoint() {
+		return "http://localhost:8081/api/v1";
+	}
+
+	@Bean
+	Move2KubeTask move2KubeTask() {
+		return new Move2KubeTask(getMove2KubeAPIEndpoint());
+	}
+
+	@Bean
+	Move2KubeRetrieve move2KubeRetrieve() {
+		return new Move2KubeRetrieve(getMove2KubeAPIEndpoint());
+	}
+
+	@Bean
+	TransformChecker transformChecker() {
+		return new TransformChecker(getMove2KubeAPIEndpoint());
+	}
+
+	@Bean(name = "transformWorkFlowChecker")
+	@Checker(cronExpression = "*/5 * * * * ?")
```

Comment:
How much time transformation is taking? Do you think checking it so often make sense?

Reply:
It's a human process, so can take a lot, but for demo purposes is great.

Reply:
yes, is the only way.